### DISCUSSION TITLE
Analysis Report - clarification required

### DISCUSSION THREAD

**Discussion by sunithakaslay** [green-code-initiative/ecoCode-csharp#67] [Q&A]
Hello All,
Thank you for the wonderful contributions being made to improve software eco score.  I needed a clarification on the analysis report.  Is there a separate analysis report created, apart from the default SonarQube analysis report? Or is it just one consolidated analysis of the default scanner and the plugin?

**Comment by vdebellabre** [comment]
Hello,
Sorry for missing your question. This repository should be closer to your subject : https://github.com/green-code-initiative/ecoCode-csharp-sonarqube

In any case, ecoCode analysis doesn't generate an additional report in Sonar. Our process uses the Sonar scanner to fetch the ecoCode rules alerts, then a custom part enriches them with severity/description/category/etc before integrating them in your regular analysis report.
